src/wallet/walletdb.cpp

```diff
@@ -924,7 +924,7 @@ bool WalletBatch::VerifyEnvironment(const fs::path& wallet_path, bilingual_str&
 
 bool WalletBatch::VerifyDatabaseFile(const fs::path& wallet_path, std::vector<bilingual_str>& warnings, bilingual_str& errorStr)
 {
-    return BerkeleyBatch::VerifyDatabaseFile(wallet_path, warnings, errorStr, WalletBatch::Recover);
```

Comment:
`WalletBatch::Recover` two-argument version is unused as of commit: 1372aba35fa91dd2fb4103ed7081a057f9794564

Reply:
Nevermind, I see you got it in 61a1e5702d0dfd3cfb6d4deb28df516a7243e196